### PR TITLE
Origin/feature/update to node18

### PR DESCRIPTION
**Pull request by ellrub**
Update from node 16 to node 18
Update firebase tools from 4.1.1 to 4.5.0
Update README.md to reflect recent changes to the repo, and also add a few missing steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,11 @@
 ### Getting started
 
 - Clone the project.
-- Change directory into `/functions`.
 - Run `npm i`.
-- Run `npm start` or `npm run serve`. Can be ran from either root or inside `/functions` directory.
+- You also need Firebase Tools before you can run this.
+    - `npm install firebase-tools`
```
Comment (nightzirch) [inline README.md]: -g, no?

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "url": "https://github.com/nightzirch/secret-toymaker.backend.git"
   },
   "engines": {
-    "node": "16"
+    "node": "18"
```
Comment (nightzirch) [inline package.json:34]: Lovely!

**Comment by nightzirch** [comment]
Looks good! Is it possible to update the line in the readme where it says node V10?

**Comment by ellrub** [comment]
Done, also missed node version in the other package json

**Review by nightzirch — APPROVED** [review]
Looks good!

**Pull request merged**